MultimodalQnA/docker_compose/intel/hpu/gaudi/README.md

```diff
@@ -339,7 +357,7 @@ curl -X POST \
     ${DATAPREP_DELETE_FILE_ENDPOINT}
 ```
 
-8. MegaService
+9. MegaService
 
```

Comment:
I would add a megaservice curl with `modalities` audio/text to demonstrate how that's used to get an audio (and text) response.